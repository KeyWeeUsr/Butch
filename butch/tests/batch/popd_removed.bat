cd %temp%
echo %errorlevel%
echo %cd%
mkdir butch-tmp-folder
echo %errorlevel%
echo %cd%
pushd butch-tmp-folder
echo %errorlevel%
echo %cd%
pushd ..
echo %errorlevel%
echo %cd%
rmdir butch-tmp-folder
echo %errorlevel%
echo %cd%
popd
echo %errorlevel%
echo %cd%
popd
echo %errorlevel%
echo %cd%
