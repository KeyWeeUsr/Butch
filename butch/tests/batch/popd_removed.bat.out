0
/tmp
0
/tmp
0
/tmp/butch-tmp-folder
0
/tmp
0
/tmp
0
/tmp
0
/tmp
